stablehlo/dialect/TypeInference.cpp

```diff
@@ -1204,8 +1223,6 @@ LogicalResult validateScatterDimensionNumbers(
     return emitOptionalError(loc,
                              "Expects update_window_dims to be sorted; got: [",
                              updateWindowDims, "].");
-
-  // scatter_c7
```

Comment:
I just followed the convension in gather, where we have one comment for both the sorted and unique if statements, since it's the same constraint.

Should I add above each if for both?